onedal/linear_model/tests/test_logistic_regression.py

```diff
@@ -38,6 +38,13 @@ def test_breast_cancer(queue, dtype):
         model = LogisticRegression(fit_intercept=True, solver="newton-cg")
         model.fit(X_train, y_train, queue=queue)
         y_pred = model.predict(X_test, queue=queue)
+
+        # TODO: check why predictions all the same on windows GPU
```

Comment:
Send me a link to the ticket associated with the TODO, and remove the TODO.

Reply:
hopefully resolved by https://github.com/intel/scikit-learn-intelex/pull/1851, removing skip to check